prosody/Dockerfile

```diff
@@ -1,10 +1,25 @@
 FROM jitsi/base
 
+ADD https://packages.prosody.im/debian/pool/main/p/prosody/prosody_0.11.1-1~stretch1_amd64.deb /tmp/prosody.deb
+
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -y lua5.2 liblua5.2-dev lua-expat lua-socket lua-filesystem ssl-cert luarocks gcc libssl1.1 libssl1.0-dev git && \
```

Comment:
Why is git necessary?

Reply:
For me it seems like git is used to download `luajwt`.

````
Installing https://luarocks.org/luajwtjitsi-1.3-7.rockspec
Cloning into 'luajwt'...
Note: checking out 'e7a6ca8cbe5941638db34e280eff7e75c07c4ec9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>
````

Reply:
👍